WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -467,11 +486,13 @@ decisions = {
 			OR = {
 				religion_group = christian
 				religion_group = pagan_group
+				has_romance_culture_trigger = yes
+				culture = roman
+				culture_group = byzantine
 			}
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
-			has_romance_culture_trigger = yes
```

Comment:
Maybe you consider the mod philosophy to be as stupid as it is unrealistic, but it will remain.